### PR TITLE
feat: add an example for cloudflare-workers

### PR DESCRIPTION
**Pull request by ogadra**
Related #24 

Hello.
I added an example of #24 using Cloudflare Workers.

https://github.com/yusukebe/hono-remix-adapter/pull/24#issuecomment-2487074456

## Screenshot

image

### CODE REVIEW

File: examples/cloudflare-workers/server/index.ts [hunk 1]
```diff
@@ -4,10 +4,14 @@ import { Hono } from 'hono'
...
     MY_VAR: string
-  }
+  },
+  Variables: {
+    'hono-context': string
+  },
 }>()
 
 app.use(async(c, next) => {
+  c.set('hono-context', 'hono-context')
```
Comment (ogadra) [inline examples/cloudflare-workers/server/index.ts:14]: I need to change the variable name.

Comment (yusukebe) [reply examples/cloudflare-workers/server/index.ts:14]: I think it's better to use the same names with the Pages example: https://github.com/yusukebe/hono-remix-adapter/pull/24/files#diff-1a16d957977e7aaf27b0228dd216f0d8e30696f730d89b6d0d03c1c45fc3babbR14

**Comment by yusukebe** [comment]
@ogadra 

Oops. This PR was closed since #24 was merged. Can you create another PR?

**Comment by ogadra** [comment]
@yusukebe I created #29 !

**Comment by yusukebe** [comment]
@ogadra Thanks!

**Pull request closed without merging**